### PR TITLE
Update _redirects

### PR DESCRIPTION
**Pull request by jnwright**
- removed dual redirects
- added /agent/dns.html redirect

### CODE REVIEW

File: website/_redirects [hunk 1]
```diff
@@ -11,16 +11,8 @@
 /segmentation                                        /use-cases/multi-platform-service-mesh            301!
 /segmentation.html                                   /use-cases/multi-platform-service-mesh            301!
 /api.html                                            /api-docs                                         301!
-/docs/agent/acl-rules.html                           /docs/acl/acl-rules                               301!
-/docs/agent/acl-rules                                /docs/acl/acl-rules                               301!
-/docs/agent/acl-system.html                          /docs/acl/acl-system                              301!
-/docs/agent/acl-system                               /docs/acl/acl-system                              301!
```
Comment (mikemorris) [inline website/_redirects:17]: I think we want to keep these origins, but update the destinations to be direct redirects? Like
```
/docs/agent/acl-rules.html                           /docs/security/acl/acl-rules                               301!
/docs/agent/acl-rules                                /docs/security/acl/acl-rules                               301!
/docs/agent/acl-system.html                          /docs/security/acl/acl-system                              301!
/docs/agent/acl-system                               /docs/security/acl/acl-system                              301!
```

File: website/_redirects [hunk 2]
```diff
@@ -11,16 +11,8 @@
...
 /docs/agent/http.html                                /api-docs                                         301!
 /docs/agent/http                                     /api-docs                                         301!
-/docs/guides/acl-legacy.html                         /docs/acl/acl-legacy                              301!
-/docs/guides/acl-legacy                              /docs/acl/acl-legacy                              301!
-/docs/guides/acl-migrate-tokens.html                 /docs/acl/acl-migrate-tokens                      301!
-/docs/guides/acl-migrate-tokens                      /docs/acl/acl-migrate-tokens                      301!
```
Comment (mikemorris) [inline website/_redirects:23]:
```
/docs/guides/acl-legacy.html                         /docs/security/acl/acl-legacy                              301!
/docs/guides/acl-legacy                              /docs/security/acl/acl-legacy                              301!
/docs/guides/acl-migrate-tokens.html                 /docs/security/acl/acl-migrate-tokens                      301!
/docs/guides/acl-migrate-tokens                      /docs/security/acl/acl-migrate-tokens                      301!
```

File: website/_redirects [hunk 3]
```diff
@@ -45,9 +37,13 @@
 /docs/internals/security                             /docs/security                                    301!
 /docs/acl/                                           /docs/security/acl/                               301!
 /docs/acl/acl-system                                 /docs/security/acl/acl-system                     301!
-/docs/acl/acl-rules                                  /docs/security/acl/acl-rules                       301!
+/docs/agent/acl-system.html                          /docs/security/acl/acl-system                     301!
```
Comment (mikemorris) [inline website/_redirects:40]:
```suggestion
```
Suggested adding this one back grouped with the other `/docs/agent` redirects in a prior comment.

**Review by mikemorris — CHANGES_REQUESTED** [review]
Thanks for updating this quick! Left a few comments because I think we want to update some of the old redirects instead of just removing them? @jescalan might know better than me.

File: website/_redirects (hunk 1 above)
Comment (jnwright) [reply website/_redirects:17]: I added the direct redirects on lines 39-46

**Comment by jnwright** [comment]
going to redo this

**Pull request closed without merging**